cl/api/views.py

```diff
@@ -198,30 +179,17 @@ async def coverage_data(request, version, court):
     else:
         court_str = "all"
     q = request.GET.get("q")
-
-    @sync_to_async
-    def query_facets(c_str, q_str):
-        with Session() as session:
-            si = ExtraSolrInterface(
-                settings.SOLR_OPINION_URL, http_connection=session, mode="r"
-            )
-            facet_field = "dateFiled"
-            return facet_field, (
-                si.query()
-                .add_extra(**build_coverage_query(c_str, q_str, facet_field))
```

Comment:
Should we remove the `build_coverage_query` and `build_court_count_query` functions in this PR?

Reply:
Yeah, removed these methods.